### ISSUE TITLE
S/F Status Flag

### ISSUE THREAD

**Issue by cybertza** [normaldotcom/canable2-fw#3]
Is there a Flag to pull status?

aka a H or S or F?

aka S =>
S31 -> Speed 3 On
S30 -> Speed 3 OFF

F ->
F0 -> OFF\Closed
F1  -> On\Open

**Comment by Elmue** [comment]
Why do you need this?
You yourself set the status and then you forget what you have set before and ask the firmware because it has a better memory than you?

**Comment by cybertza** [comment]
Yes, obviously one may not want to set it all the time as it may lead to storage rot, but reading a register does not. Furthermore external devices may reboot independently from runtime so you cannot guarantee a single set and forget either. 

Functional vs practical vs good practice.

**Comment by Elmue** [comment]
I have no idea what you are talking about.
All these settings are stored in RAM.
How will this lead to storage rot ????

It seems you live under the wrong assumption that these settings are stored in flash memory.
You are totally wrong.

> Furthermore external devices may reboot independently 

After a reboot or a disconnection of the USB cable the CANable has forgotten all your settings and you must sent them again.
Each time you open the CANable you execute the same commands to configure the device:

- Close the device in case it is still open (you never can be sure)
- Set the baudrate(s)
- Set any other settings that you need like for example silent mode
- Open the device
